File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        
        String[] num = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        
        for (int i = 0; i < num.length; i++) {
            s = s.replace(num[i], String.valueOf(i));
        }
        
        answer = Integer.parseInt(s);
        return answer;
    }
}

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    var answer = 0;
    
    let num = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
    
    for (let i in num) {
        s = s.replaceAll(num[i], i.toString());
    }
    
    answer = Number(s);
    
    return answer;
}